src/main/java/ludum/dare/components/PowerUpComponents/StealCoinsComponent.java

```diff
@@ -1,19 +1,19 @@
 package ludum.dare.components.PowerUpComponents;
 
+import ludum.dare.components.PlayerCurrencyComponent;
+import ludum.dare.components.TimedComponent;
 import ludum.dare.interfaces.IComponent;
 import ludum.dare.interfaces.IUpdate;
 
 /**
  * Created by jake on 12/12/2015.
  */
-public class StealCoinsComponent implements IComponent, IUpdate{
+public class StealCoinsComponent implements IComponent{
 
-    public StealCoinsComponent(){
-
-    }
-
-    @Override
-    public void update(float delta) {
-        
+    public StealCoinsComponent(PlayerCurrencyComponent myWallet, PlayerCurrencyComponent aWallet, PlayerCurrencyComponent bWallet, PlayerCurrencyComponent cWallet) {
+        myWallet.getACoin(3);
+        aWallet.loseACoin();
+        bWallet.loseACoin();
+        cWallet.loseACoin();
```

Comment:
Players is a static list we can get to now -- so it might make more sense to dynamically grab the players out of that list since we are not guaranteed to have 4 players in every game (maybe we are... I dunno)